crates/common/axum_tls/src/config.rs

```diff
@@ -161,7 +161,10 @@ impl PemReader for InjectedValue<String> {
 }
 
 impl<P: AsRef<Path> + Debug + ?Sized> PemReader for P {
-    type Read<'a> = File where Self: 'a;
+    type Read<'a>
+        = File
+    where
+        Self: 'a;
```

Comment:
Hmm... `cargo-fmt` pushing too far.

Reply:
Yep. And I always forget to do a `rustup update` to update all of the Rust components, so it took me awhile to actually run the required version of Rustfmt